### ISSUE TITLE
Typescript Support?

### ISSUE THREAD

**Issue by serranoarevalo** [pinterest/gestalt#275]
Is it on the roadmap? PR's accepted?

**Comment by chrislloyd** [comment]
Not currently on the roadmap. Happy to accept PRs but we don't use Typescript so you probably wouldn't receive much support from us officially. Perhaps adding definitions to DefinitelyTyped might be the best path forward?

/cc @btraut who has experience with Typescript.

**Comment by serranoarevalo** [comment]
Gonna start working on a PR for DefinitelyTyped then. Cheers!

**Comment by btraut** [comment]
Agreed with Chris. I'd _love_ to have TS type definitions for Gestalt, but I don't think we're prepared to support them. As such, I wouldn't feel comfortable colocating the `.d.ts` with the Gestalt code. Creating one in DefinitelyTyped is the place to go.

**Comment by serranoarevalo** [comment]
Done! https://github.com/DefinitelyTyped/DefinitelyTyped/pull/27279, let's wait and see 💪🏻

**Comment by serranoarevalo** [comment]
Merged!

**Comment by serranoarevalo** [comment]
https://www.npmjs.com/package/@types/gestalt

**Comment by smachs** [comment]
Working!
